src/common/init.cpp

```diff
@@ -75,6 +76,9 @@ void PelotonInit::Initialize() {
   pg_catalog->CreateDatabase(DEFAULT_DB_NAME, txn);
 
   txn_manager.CommitTransaction(txn);
```

Comment:
Is this ok to call commit, before recovery? If ok, then can you comment why?

Reply:
It was a design decision to let the catalog initialize and bootstrap before recovery. Therefore, since the catalog is always rebuilt on bootstrapping, we avoid logging those entries.